src/Php74/Php74.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Php74;
+
+/**
+ * @author Ion Bazan <ion.bazan@gmail.com>
+ *
+ * @internal
+ */
+final class Php74
+{
+    /**
+     * @param object $obj
+     *
+     * @return array
+     */
+    public static function get_mangled_object_vars($obj)
+    {
+        $result = array();
+
+        if ($obj instanceof \ArrayIterator || $obj instanceof \ArrayObject) {
+            $reflector = new \ReflectionClass($obj instanceof \ArrayIterator ? 'ArrayIterator' : 'ArrayObject');
+            $flags = $reflector->getMethod('getFlags')->invoke($obj);
+            $reflector = $reflector->getMethod('setFlags');
+
+            $reflector->invoke($obj, ($flags & \ArrayObject::STD_PROP_LIST) ? 0 : \ArrayObject::STD_PROP_LIST);
+            $result = (array) $obj;
+            $reflector->invoke($obj, $flags);
+
+            return $result;
+        }
+
+        foreach ((array) $obj as $key => $value) {
+            $result[(string) $key] = $value;
+        }
+
+        return $result;
+    }
+
+    /**
+     * @param string      $string
+     * @param int         $split_length
+     * @param string|null $encoding
+     *
+     * @return array|false
+     */
+    public static function mb_str_split($string, $split_length = 1, $encoding = null)
+    {
+        if (null === $encoding) {
+            $encoding = mb_internal_encoding();
+        }
+
+        if ($split_length < 1) {
+            trigger_error('The length of each segment must be greater than zero', E_USER_WARNING);
+            return false;
+        }
+
+        $result = array();
+        $length = mb_strlen($string, $encoding);
+
+        for ($i = 0; $i < $length; $i += $split_length) {
+            $result[] = mb_substr($string, $i, $split_length, $encoding);
+        }
+
+        return $result;
+    }
+
+    /**
+     * @return array
+     */
+    public static function password_algos()
+    {
+        $algos = array(PASSWORD_BCRYPT);
```

Comment:
this one too should be under an "if" statement, as the polyfill runs on PHP< 5.5, isn't it?

Reply:
Actually the test was passing because we've had the polyfill bootstapped. I added the `if` anyway.